src/app/static/src/tests/surveyAndProgram/mutations.test.ts

```diff
@@ -6,21 +6,22 @@ import {mutations} from "../../app/store/surveyAndProgram/mutations";
 import { mockSurveyResponse} from "../mocks";
 import {initialBaselineState} from "../../app/store/baseline/baseline";
 import {Module} from "vuex";
-import {RootState} from "../../app/main";
+import {RootState} from "../../app/root";
+import {initialFilteredDataState} from "../../app/store/filteredData/filteredData";
 
 describe("Survey and program mutations", () => {
 
     const testPayload = {
         payload: mockSurveyResponse({
-            data: "SOME DATA",
+            data: [],
             filename: "somefile.csv"
         })
     };
 
     it("sets survey data and filename on SurveyLoaded", () => {
         const testState = {...initialSurveyAndProgramDataState};
         mutations.SurveyLoaded(testState, testPayload);
-        expect(testState.survey!!.data).toBe("SOME DATA");
+        expect(testState.survey!!.data.length).toBe(0);
```

Comment:
Feel like this check is kind of pointless, since many things could satisfy it. Can we leave these tests as they were and just use `data: "SOME DATA" as any` here: https://github.com/mrc-ide/hint/pull/43/files#diff-463a72978089f680367017681d7a669bL15